### PR TITLE
LPS-121567 Update to @liferay/npm-scripts v33.0.2

### PR DESCRIPTION
**Pull request by wincent**
- [v33.0.0 release notes](https://github.com/liferay/liferay-frontend-projects/releases/tag/npm-scripts%2Fv33.0.0)
- [v33.0.1 release notes](https://github.com/liferay/liferay-frontend-projects/releases/tag/npm-scripts%2Fv33.0.1)
- [v33.0.2 release notes](https://github.com/liferay/liferay-frontend-projects/releases/tag/npm-scripts%2Fv33.0.2)

This is a "refactor" release which should not change functionality. We've consolidated eslint-config-liferay and all of the liferay-npm-tools packages into a single monorepo (with more to come, to reduce the friction of making cross-cutting changes, coordinating releases, and doing maintenance in general).

As part of the move, we are moving the packages under the `@liferay/` named scope on npm, which gives us a centralized namespace which we control and can audit, manage access etc.

This is a major version bump because it is a breaking change in the sense that:

- All the references to the `liferay-npm-scripts` package need to be changed (to `@liferay/npm-scripts`; although note that the executable itself is still called `liferay-npm-scripts`).
- All of our custom ESLint rules are now in named scopes too, so those need to be updated as well.

One minor thing: I had to add "acorn" explicitly to the top level to force Yarn's hoisting algorithm to put it at the top, where ESLint needs it. In a future update we'll be able to remove this by updating other dependencies that also use acorn. I'm not doing that now because I want this switch from un-scoped to named-scope to be minimal, and because there is an open issue in the Jest repo about a performance regression over there that makes it undesirable for us to update just yet.

More details in comments on [this PR](https://github.com/liferay/liferay-frontend-projects/pull/85).

[Jest issue](https://github.com/facebook/jest/issues/9457).

### CODE REVIEW

**Comment by wincent** [comment]
ci:test:sf

**Comment by wincent** [comment]
ci:test:relevant

**Comment by wincent** [comment]
Bah, I see I need one more tweak to make JS unit tests pass (they still reference the old reporter).

**Comment by wincent** [comment]
Fix for the JS unit tests is: https://github.com/liferay/liferay-frontend-projects/pull/89

**Review by jbalsas — APPROVED** [review]
LGTM! Please be greeN! :D

**Comment by wincent** [comment]
not-easy-being-green

**Comment by wincent** [comment]
ci:stop

**Comment by wincent** [comment]
ci:test:sf

**Comment by wincent** [comment]
ci:test:relevant

**Comment by wincent** [comment]
Well:

```
A problem occurred evaluating project ':apps:frontend-editor:frontend-editor-ckeditor-web'.
     [exec] > Ambiguous method overloading for method java.io.File#<init>.
     [exec]   Cannot resolve which method to invoke for [null, class java.lang.String] due to overlapping prototypes between:
     [exec]   	[class java.lang.String, class java.lang.String]
     [exec]   	[class java.io.File, class java.lang.String]
```

That seems obviously unrelated. Any thoughts @john-co, @PablitoBonito?

**Comment by wincent** [comment]
Going to re-run in the meantime to see if I get lucky.

**Comment by wincent** [comment]
ci:test:relevant

**Comment by bryceosterhaus** [comment]
The `@liferay/` namespace looks so clean 🔥

**Comment by wincent** [comment]
Ok, seeing as CI is not being helpful, I'm going to rebase and see if that makes a difference.

**Comment by wincent** [comment]
ci:test:relevant

**Comment by wincent** [comment]
ci:test:sf

**Comment by wincent** [comment]
ci:test:bundle

**Comment by john-co** [comment]
I haven't seen this error in other PRs today, ie: https://github.com/john-co/liferay-portal/pull/347#issuecomment-700918835

I'm leaning towards that this error is related to the PR. I'm fetching the branch to investigate further.

**Comment by john-co** [comment]
After fetching the branch and running `ant all`, I'm able to reproduce error locally:
```
war:
     [echo] Updating /home/john-co/Liferay/public/liferay-binaries-cache-2020.
     [exec] Already up to date.From https://github.com/liferay/liferay-binaries-cache-2020
     [exec]  * branch              HEAD       -> FETCH_HEAD
     [exec] 
     [exec] Current branch master is up to date.
 [sync-dir] Synchronizing /home/john-co/Liferay/public/liferay-binaries-cache-2020/.gradle/caches/modules-2/files-2.1 into /home/john-co/Liferay/public/liferay-portal/.gradle/caches/modules-2/files-2.1
 [sync-dir] 0 files synchronized in 301ms
     [echo] Executing Gradle task: deploy
     [echo] Executing Gradle task: deploy
     [echo] FAILURE: Build failed with an exception.
     [echo] 
     [echo] * Where:
     [echo] Build file '/home/john-co/Liferay/public/liferay-portal/modules/apps/frontend-editor/frontend-editor-ckeditor-web/build.gradle' line: 53
     [echo] 
     [echo] * What went wrong:
     [echo] A problem occurred evaluating project ':apps:frontend-editor:frontend-editor-ckeditor-web'.
     [echo] > Ambiguous method overloading for method java.io.File#<init>.
     [echo]   Cannot resolve which method to invoke for [null, class java.lang.String] due to overlapping prototypes between:
     [echo]   	[class java.lang.String, class java.lang.String]
     [echo]   	[class java.io.File, class java.lang.String]
     [echo] 
     [echo] * Try:
     [echo] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [echo] 
     [echo] * Exception is:
     [echo] org.gradle.api.GradleScriptException: A problem occurred evaluating project ':apps:frontend-editor:frontend-editor-ckeditor-web'.
     [echo] 	at org.gradle.groovy.scripts.internal.DefaultScriptRunnerFactory$ScriptRunnerImpl.run(DefaultScriptRunnerFactory.java:93)
     [echo] 	at org.gradle.configuration.DefaultScriptPluginFactory$ScriptPluginImpl$2.run(DefaultScriptPluginFactory.java:227)
     [echo] 	at org.gradle.configuration.ProjectScriptTarget.addConfiguration(ProjectScriptTarget.java:77)
     [echo] 	at org.gradle.configuration.DefaultScriptPluginFactory$ScriptPluginImpl.apply(DefaultScriptPluginFactory.java:232)
     [echo] 	at org.gradle.configuration.BuildOperationScriptPlugin$1$1.run(BuildOperationScriptPlugin.java:69)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:402)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:394)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$1.execute(DefaultBuildOperationExecutor.java:165)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:250)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:158)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:92)
     [echo] 	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
     [echo] 	at org.gradle.configuration.BuildOperationScriptPlugin$1.execute(BuildOperationScriptPlugin.java:66)
     [echo] 	at org.gradle.configuration.BuildOperationScriptPlugin$1.execute(BuildOperationScriptPlugin.java:63)
     [echo] 	at org.gradle.configuration.internal.DefaultUserCodeApplicationContext.apply(DefaultUserCodeApplicationContext.java:49)
     [echo] 	at org.gradle.configuration.BuildOperationScriptPlugin.apply(BuildOperationScriptPlugin.java:63)
     [echo] 	at org.gradle.configuration.project.BuildScriptProcessor$1.run(BuildScriptProcessor.java:45)
     [echo] 	at org.gradle.internal.Factories$1.create(Factories.java:26)
     [echo] 	at org.gradle.api.internal.project.DefaultProjectStateRegistry$ProjectStateImpl.withMutableState(DefaultProjectStateRegistry.java:193)
     [echo] 	at org.gradle.api.internal.project.DefaultProjectStateRegistry$ProjectStateImpl.withMutableState(DefaultProjectStateRegistry.java:187)
     [echo] 	at org.gradle.configuration.project.BuildScriptProcessor.execute(BuildScriptProcessor.java:42)
     [echo] 	at org.gradle.configuration.project.BuildScriptProcessor.execute(BuildScriptProcessor.java:26)
     [echo] 	at org.gradle.configuration.project.ConfigureActionsProjectEvaluator.evaluate(ConfigureActionsProjectEvaluator.java:35)
     [echo] 	at org.gradle.configuration.project.LifecycleProjectEvaluator$EvaluateProject$1.run(LifecycleProjectEvaluator.java:107)
     [echo] 	at org.gradle.internal.Factories$1.create(Factories.java:26)
     [echo] 	at org.gradle.api.internal.project.DefaultProjectStateRegistry$ProjectStateImpl.withMutableState(DefaultProjectStateRegistry.java:193)
     [echo] 	at org.gradle.api.internal.project.DefaultProjectStateRegistry$ProjectStateImpl.withMutableState(DefaultProjectStateRegistry.java:187)
     [echo] 	at org.gradle.configuration.project.LifecycleProjectEvaluator$EvaluateProject.run(LifecycleProjectEvaluator.java:96)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:402)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:394)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$1.execute(DefaultBuildOperationExecutor.java:165)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:250)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:158)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:92)
     [echo] 	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
     [echo] 	at org.gradle.configuration.project.LifecycleProjectEvaluator.evaluate(LifecycleProjectEvaluator.java:68)
     [echo] 	at org.gradle.api.internal.project.DefaultProject.evaluate(DefaultProject.java:693)
     [echo] 	at org.gradle.api.internal.project.DefaultProject.evaluate(DefaultProject.java:141)
     [echo] 	at org.gradle.execution.TaskPathProjectEvaluator.configure(TaskPathProjectEvaluator.java:36)
     [echo] 	at org.gradle.execution.TaskPathProjectEvaluator.configureHierarchy(TaskPathProjectEvaluator.java:64)
     [echo] 	at org.gradle.execution.TaskSelector.getSelection(TaskSelector.java:101)
     [echo] 	at org.gradle.execution.TaskSelector.getSelection(TaskSelector.java:82)
     [echo] 	at org.gradle.execution.commandline.CommandLineTaskParser.parseTasks(CommandLineTaskParser.java:42)
     [echo] 	at org.gradle.execution.TaskNameResolvingBuildConfigurationAction.configure(TaskNameResolvingBuildConfigurationAction.java:46)
     [echo] 	at org.gradle.execution.DefaultBuildConfigurationActionExecuter.configure(DefaultBuildConfigurationActionExecuter.java:58)
     [echo] 	at org.gradle.execution.DefaultBuildConfigurationActionExecuter.access$200(DefaultBuildConfigurationActionExecuter.java:26)
     [echo] 	at org.gradle.execution.DefaultBuildConfigurationActionExecuter$2.proceed(DefaultBuildConfigurationActionExecuter.java:66)
     [echo] 	at org.gradle.execution.DefaultTasksBuildExecutionAction.configure(DefaultTasksBuildExecutionAction.java:45)
     [echo] 	at org.gradle.execution.DefaultBuildConfigurationActionExecuter.configure(DefaultBuildConfigurationActionExecuter.java:58)
     [echo] 	at org.gradle.execution.DefaultBuildConfigurationActionExecuter.access$200(DefaultBuildConfigurationActionExecuter.java:26)
     [echo] 	at org.gradle.execution.DefaultBuildConfigurationActionExecuter$2.proceed(DefaultBuildConfigurationActionExecuter.java:66)
     [echo] 	at org.gradle.execution.ExcludedTaskFilteringBuildConfigurationAction.configure(ExcludedTaskFilteringBuildConfigurationAction.java:48)
     [echo] 	at org.gradle.execution.DefaultBuildConfigurationActionExecuter.configure(DefaultBuildConfigurationActionExecuter.java:58)
     [echo] 	at org.gradle.execution.DefaultBuildConfigurationActionExecuter.access$200(DefaultBuildConfigurationActionExecuter.java:26)
     [echo] 	at org.gradle.execution.DefaultBuildConfigurationActionExecuter$1.run(DefaultBuildConfigurationActionExecuter.java:44)
     [echo] 	at org.gradle.internal.Factories$1.create(Factories.java:26)
     [echo] 	at org.gradle.api.internal.project.DefaultProjectStateRegistry.withLenientState(DefaultProjectStateRegistry.java:133)
     [echo] 	at org.gradle.api.internal.project.DefaultProjectStateRegistry.withLenientState(DefaultProjectStateRegistry.java:125)
     [echo] 	at org.gradle.execution.DefaultBuildConfigurationActionExecuter.select(DefaultBuildConfigurationActionExecuter.java:40)
     [echo] 	at org.gradle.initialization.DefaultTaskExecutionPreparer.prepareForTaskExecution(DefaultTaskExecutionPreparer.java:38)
     [echo] 	at org.gradle.initialization.BuildOperatingFiringTaskExecutionPreparer$CalculateTaskGraph.populateTaskGraph(BuildOperatingFiringTaskExecutionPreparer.java:82)
     [echo] 	at org.gradle.initialization.BuildOperatingFiringTaskExecutionPreparer$CalculateTaskGraph.run(BuildOperatingFiringTaskExecutionPreparer.java:57)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:402)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:394)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$1.execute(DefaultBuildOperationExecutor.java:165)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:250)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:158)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:92)
     [echo] 	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
     [echo] 	at org.gradle.initialization.BuildOperatingFiringTaskExecutionPreparer.prepareForTaskExecution(BuildOperatingFiringTaskExecutionPreparer.java:45)
     [echo] 	at org.gradle.initialization.DefaultGradleLauncher.prepareTaskExecution(DefaultGradleLauncher.java:206)
     [echo] 	at org.gradle.initialization.DefaultGradleLauncher.doClassicBuildStages(DefaultGradleLauncher.java:142)
     [echo] 	at org.gradle.initialization.DefaultGradleLauncher.doBuildStages(DefaultGradleLauncher.java:126)
     [echo] 	at org.gradle.initialization.DefaultGradleLauncher.executeTasks(DefaultGradleLauncher.java:106)
     [echo] 	at org.gradle.internal.invocation.GradleBuildController$1.execute(GradleBuildController.java:60)
     [echo] 	at org.gradle.internal.invocation.GradleBuildController$1.execute(GradleBuildController.java:57)
     [echo] 	at org.gradle.internal.invocation.GradleBuildController$3.create(GradleBuildController.java:85)
     [echo] 	at org.gradle.internal.invocation.GradleBuildController$3.create(GradleBuildController.java:78)
     [echo] 	at org.gradle.internal.work.DefaultWorkerLeaseService.withLocks(DefaultWorkerLeaseService.java:189)
     [echo] 	at org.gradle.internal.work.StopShieldingWorkerLeaseService.withLocks(StopShieldingWorkerLeaseService.java:40)
     [echo] 	at org.gradle.internal.invocation.GradleBuildController.doBuild(GradleBuildController.java:78)
     [echo] 	at org.gradle.internal.invocation.GradleBuildController.run(GradleBuildController.java:57)
     [echo] 	at org.gradle.tooling.internal.provider.ExecuteBuildActionRunner.run(ExecuteBuildActionRunner.java:31)
     [echo] 	at org.gradle.launcher.exec.ChainingBuildActionRunner.run(ChainingBuildActionRunner.java:35)
     [echo] 	at org.gradle.launcher.exec.BuildOutcomeReportingBuildActionRunner.run(BuildOutcomeReportingBuildActionRunner.java:63)
     [echo] 	at org.gradle.tooling.internal.provider.ValidatingBuildActionRunner.run(ValidatingBuildActionRunner.java:32)
     [echo] 	at org.gradle.launcher.exec.BuildCompletionNotifyingBuildActionRunner.run(BuildCompletionNotifyingBuildActionRunner.java:39)
     [echo] 	at org.gradle.launcher.exec.RunAsBuildOperationBuildActionRunner$3.call(RunAsBuildOperationBuildActionRunner.java:51)
     [echo] 	at org.gradle.launcher.exec.RunAsBuildOperationBuildActionRunner$3.call(RunAsBuildOperationBuildActionRunner.java:45)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$CallableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:416)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$CallableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:406)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$1.execute(DefaultBuildOperationExecutor.java:165)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:250)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:158)
     [echo] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:102)
     [echo] 	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.call(DelegatingBuildOperationExecutor.java:36)
     [echo] 	at org.gradle.launcher.exec.RunAsBuildOperationBuildActionRunner.run(RunAsBuildOperationBuildActionRunner.java:45)
     [echo] 	at org.gradle.launcher.exec.InProcessBuildActionExecuter$1.transform(InProcessBuildActionExecuter.java:50)
     [echo] 	at org.gradle.launcher.exec.InProcessBuildActionExecuter$1.transform(InProcessBuildActionExecuter.java:47)
     [echo] 	at org.gradle.composite.internal.DefaultRootBuildState.run(DefaultRootBuildState.java:78)
     [echo] 	at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:47)
     [echo] 	at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:31)
     [echo] 	at org.gradle.launcher.exec.BuildTreeScopeBuildActionExecuter.execute(BuildTreeScopeBuildActionExecuter.java:42)
     [echo] 	at org.gradle.launcher.exec.BuildTreeScopeBuildActionExecuter.execute(BuildTreeScopeBuildActionExecuter.java:28)
     [echo] 	at org.gradle.tooling.internal.provider.ContinuousBuildActionExecuter.execute(ContinuousBuildActionExecuter.java:78)
     [echo] 	at org.gradle.tooling.internal.provider.ContinuousBuildActionExecuter.execute(ContinuousBuildActionExecuter.java:52)
     [echo] 	at org.gradle.tooling.internal.provider.SubscribableBuildActionExecuter.execute(SubscribableBuildActionExecuter.java:59)
     [echo] 	at org.gradle.tooling.internal.provider.SubscribableBuildActionExecuter.execute(SubscribableBuildActionExecuter.java:36)
     [echo] 	at org.gradle.tooling.internal.provider.SessionScopeBuildActionExecuter.execute(SessionScopeBuildActionExecuter.java:68)
     [echo] 	at org.gradle.tooling.internal.provider.SessionScopeBuildActionExecuter.execute(SessionScopeBuildActionExecuter.java:38)
     [echo] 	at org.gradle.tooling.internal.provider.GradleThreadBuildActionExecuter.execute(GradleThreadBuildActionExecuter.java:37)
     [echo] 	at org.gradle.tooling.internal.provider.GradleThreadBuildActionExecuter.execute(GradleThreadBuildActionExecuter.java:26)
     [echo] 	at org.gradle.tooling.internal.provider.ParallelismConfigurationBuildActionExecuter.execute(ParallelismConfigurationBuildActionExecuter.java:43)
     [echo] 	at org.gradle.tooling.internal.provider.ParallelismConfigurationBuildActionExecuter.execute(ParallelismConfigurationBuildActionExecuter.java:29)
     [echo] 	at org.gradle.tooling.internal.provider.StartParamsValidatingActionExecuter.execute(StartParamsValidatingActionExecuter.java:60)
     [echo] 	at org.gradle.tooling.internal.provider.StartParamsValidatingActionExecuter.execute(StartParamsValidatingActionExecuter.java:32)
     [echo] 	at org.gradle.tooling.internal.provider.SessionFailureReportingActionExecuter.execute(SessionFailureReportingActionExecuter.java:55)
     [echo] 	at org.gradle.tooling.internal.provider.SessionFailureReportingActionExecuter.execute(SessionFailureReportingActionExecuter.java:41)
     [echo] 	at org.gradle.tooling.internal.provider.SetupLoggingActionExecuter.execute(SetupLoggingActionExecuter.java:48)
     [echo] 	at org.gradle.tooling.internal.provider.SetupLoggingActionExecuter.execute(SetupLoggingActionExecuter.java:32)
     [echo] 	at org.gradle.launcher.daemon.server.exec.ExecuteBuild.doBuild(ExecuteBuild.java:68)
     [echo] 	at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)
     [echo] 	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
     [echo] 	at org.gradle.launcher.daemon.server.exec.WatchForDisconnection.execute(WatchForDisconnection.java:39)
     [echo] 	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
     [echo] 	at org.gradle.launcher.daemon.server.exec.ResetDeprecationLogger.execute(ResetDeprecationLogger.java:27)
     [echo] 	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
     [echo] 	at org.gradle.launcher.daemon.server.exec.RequestStopIfSingleUsedDaemon.execute(RequestStopIfSingleUsedDaemon.java:35)
     [echo] 	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
     [echo] 	at org.gradle.launcher.daemon.server.exec.ForwardClientInput$2.create(ForwardClientInput.java:78)
     [echo] 	at org.gradle.launcher.daemon.server.exec.ForwardClientInput$2.create(ForwardClientInput.java:75)
     [echo] 	at org.gradle.util.Swapper.swap(Swapper.java:38)
     [echo] 	at org.gradle.launcher.daemon.server.exec.ForwardClientInput.execute(ForwardClientInput.java:75)
     [echo] 	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
     [echo] 	at org.gradle.launcher.daemon.server.exec.LogAndCheckHealth.execute(LogAndCheckHealth.java:55)
     [echo] 	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
     [echo] 	at org.gradle.launcher.daemon.server.exec.LogToClient.doBuild(LogToClient.java:63)
     [echo] 	at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)
     [echo] 	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
     [echo] 	at org.gradle.launcher.daemon.server.exec.EstablishBuildEnvironment.doBuild(EstablishBuildEnvironment.java:82)
     [echo] 	at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)
     [echo] 	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
     [echo] 	at org.gradle.launcher.daemon.server.exec.StartBuildOrRespondWithBusy$1.run(StartBuildOrRespondWithBusy.java:52)
     [echo] 	at org.gradle.launcher.daemon.server.DaemonStateCoordinator$1.run(DaemonStateCoordinator.java:297)
     [echo] 	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
     [echo] 	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)
     [echo] 	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:56)
     [echo] Caused by: groovy.lang.GroovyRuntimeException: Ambiguous method overloading for method java.io.File#<init>.
     [echo] Cannot resolve which method to invoke for [null, class java.lang.String] due to overlapping prototypes between:
     [echo] 	[class java.lang.String, class java.lang.String]
     [echo] 	[class java.io.File, class java.lang.String]
     [echo] 	at build_2tekrqka4ni3zmg8uuu6lcmdm$_run_closure3.doCall(/home/john-co/Liferay/public/liferay-portal/modules/apps/frontend-editor/frontend-editor-ckeditor-web/build.gradle:53)
     [echo] 	at org.gradle.util.ClosureBackedAction.execute(ClosureBackedAction.java:71)
     [echo] 	at org.gradle.util.ConfigureUtil.configureTarget(ConfigureUtil.java:154)
     [echo] 	at org.gradle.util.ConfigureUtil.configureSelf(ConfigureUtil.java:130)
     [echo] 	at org.gradle.api.internal.AbstractTask.configure(AbstractTask.java:587)
     [echo] 	at org.gradle.api.internal.AbstractTask.configure(AbstractTask.java:91)
     [echo] 	at org.gradle.util.ConfigureUtil.configure(ConfigureUtil.java:103)
     [echo] 	at org.gradle.api.internal.DefaultNamedDomainObjectCollection$ContainerElementsDynamicObject.tryInvokeMethod(DefaultNamedDomainObjectCollection.java:541)
     [echo] 	at org.gradle.internal.metaobject.CompositeDynamicObject.tryInvokeMethod(CompositeDynamicObject.java:98)
     [echo] 	at org.gradle.internal.extensibility.MixInClosurePropertiesAsMethodsDynamicObject.tryInvokeMethod(MixInClosurePropertiesAsMethodsDynamicObject.java:33)
     [echo] 	at org.gradle.groovy.scripts.BasicScript$ScriptDynamicObject.tryInvokeMethod(BasicScript.java:134)
     [echo] 	at org.gradle.internal.metaobject.AbstractDynamicObject.invokeMethod(AbstractDynamicObject.java:162)
     [echo] 	at org.gradle.groovy.scripts.BasicScript.invokeMethod(BasicScript.java:83)
     [echo] 	at build_2tekrqka4ni3zmg8uuu6lcmdm.run(/home/john-co/Liferay/public/liferay-portal/modules/apps/frontend-editor/frontend-editor-ckeditor-web/build.gradle:48)
     [echo] 	at org.gradle.groovy.scripts.internal.DefaultScriptRunnerFactory$ScriptRunnerImpl.run(DefaultScriptRunnerFactory.java:91)
     [echo] 	... 146 more
     [echo] 
     [echo] 
     [echo] * Get more help at https://help.gradle.org
     [echo] 
     [echo] BUILD FAILED in 20s
```

**Comment by john-co** [comment]
Not able to reproduce issue after reverting LPS-121567.

For the error, it looks like a `null` is being passed in somewhere causing some confusion as to which constructor to use during the gradle deploy task...but I can't pinpoint the line of code that's exactly causing that atm...

**Comment by wincent** [comment]
Ok, thanks for looking into it @john-co. I'll dig in deeper tomorrow.

**Comment by wincent** [comment]
I think these two fixes should do it:

- https://github.com/liferay/liferay-js-toolkit/pull/654
- https://github.com/liferay/liferay-frontend-projects/pull/90

**Comment by wincent** [comment]
Ok, let's give it one more try with those fixes in place (via `@liferay/npm-scripts` v33.0.2). 🤞

**Comment by wincent** [comment]
ci:test:sf

**Comment by wincent** [comment]
ci:test:relevant

**Comment by wincent** [comment]
Found a couple more references...

**Comment by wincent** [comment]
ci:stop

**Comment by wincent** [comment]
@izaera: does the Java look acceptable [in this commit](https://github.com/liferay-frontend/liferay-portal/pull/409/commits/c432642486192d6bb5ce69b6ac421973c9071a72)?

I'm not sure exactly how to push this and it has been a while since I touched a Gradle plugin, so I am just going to see if CI likes it while I go back through the history and see if anything in there jogs my memory.

**Comment by wincent** [comment]
ci:test:sf

**Comment by wincent** [comment]
ci:test:relevant

File: modules/sdk/gradle-plugins-node/src/main/java/com/liferay/gradle/plugins/node/NodePlugin.java [hunk 1]
```diff
@@ -1012,14 +1012,23 @@ private boolean _hasLiferayNpmScripts12Dependency(File packageJSONFile) {
...
+		}
+
+		String dependencyName = null;
 
+		if (devDependencies.containsKey("@liferay/npm-scripts")) {
+			dependencyName = "@liferay/npm-scripts";
+		}
+		else if (devDependencies.containsKey("@liferay/npm-scripts")) {
+			dependencyName = "liferay-npm-scripts";
+		}
```
Comment (izaera) [inline modules/sdk/gradle-plugins-node/src/main/java/com/liferay/gradle/plugins/node/NodePlugin.java:1026]: Is this wrong? :thinking: (the two ifs)

Comment (wincent) [reply modules/sdk/gradle-plugins-node/src/main/java/com/liferay/gradle/plugins/node/NodePlugin.java:1026]: You have passed the test. Well done, my young ~padawan~ kung fu fighter.

wise-man

**Comment by wincent** [comment]
ci:stop

**Comment by wincent** [comment]
ci:test:sf

**Comment by wincent** [comment]
ci:test:relevant

**Comment by wincent** [comment]
Sent a copy of this to Peter Shin to ask for his help with updating the Gradle plug-in: https://github.com/petershin/liferay-portal/pull/1213

**Comment by wincent** [comment]
ci:test:sf

**Comment by wincent** [comment]
ci:test:relevant

**Comment by wincent** [comment]
ci:test:sf

**Comment by wincent** [comment]
ci:test:relevant

**Comment by wincent** [comment]
ci:test:bundle

**Comment by wincent** [comment]
ci:forward

**Pull request closed without merging**